WordPress/Classes/Services/PostCoordinator+Dashboard.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// This extension emits notification so the posts card on
+/// the Dashboard can listen to them
+///
+/// Basically this is needed to show a card when a post is
+/// scheduled or drafted and this card is not appearing
+extension PostCoordinator {
+    func notifyDashboardOfPostScheduled() {
+        NotificationCenter.default.post(name: .showScheduledCard, object: nil)
+    }
+
+    func notifyDashboardOfDraftSaved() {
+        NotificationCenter.default.post(name: .showDraftsCard, object: nil)
+    }
+}
+
+extension NSNotification.Name {
+    /// Fired when a post is scheduled
+    static let showScheduledCard = NSNotification.Name("ShowScheduledCard")
+
+    /// Fired when a draft is saved
+    static let showDraftsCard = NSNotification.Name("ShowDraftsCard")
```

Comment:
[nit] I feel the notification name should be descriptive to what happened, not the action that should be taken after firing the notification. What do you think?
```suggestion
    static let NewPostScheduled = NSNotification.Name("NewPostScheduled")

    /// Fired when a draft is saved
    static let NewDraftPostSaved = NSNotification.Name("NewDraftPostSaved")
```

Reply:
I agree!

To be honest I added those names to try to discourage others from relying on those notifications. But we don't necessarily solve a problem by creating other, changed in 1923d558f68f8e9c5864c996de506ceef4935c7d